src/js/tracks/text-track-list-converter.js

```diff
@@ -28,8 +28,11 @@ let trackToJson_ = function(track) {
         id: cue.id
       };
     }),
-    src: track.src
   };
+  if (track.src) {
```

Comment:
Yeah, add-if-present might be more efficient (in terms of reading the code if not performance).
```js
let ret = {};
['kind', 'label', 'language', 'id', 'inBand...', 'mode', 'src'].forEach(k => {
  if (track[k]) {
    ret[k] = track[k];
  }
});
if (track.cues) {
  // special map handling for cues
}
return ret;
```